### ISSUE TITLE
Remove phan support

### ISSUE THREAD

**Issue by xepozz** [yiisoft/yii-dev-tool#107]
We was migrate from `phan` to `psalm` partially. 
I think we need to remove any `phan` mentions from packages.

**Comment by rugabarbo** [comment]
@samdark do we have a special place for infrastructure/general tasks? It looks like this task is about yiisoft development in general, but not about `yii-dev-tool` itself.

**Comment by samdark** [comment]
No, we don't. Well, Trello is more like it: https://trello.com/b/GiAnIAir/yii-3
